paymentsheet/build.gradle

```diff
@@ -21,6 +21,7 @@ if (System.getenv("JITPACK")) {
 dependencies {
     implementation project(":payments-core")
     implementation project(':payments-ui-core')
+    implementation "androidx.constraintlayout:constraintlayout-compose:1.0.0"
```

Comment:
Can we use the libraries.androidx.* for this dependency.  See this [PR](https://github.com/stripe/stripe-android/pull/4601/files) for reference.

Reply:
Should this also point to `$composeVersion`?

Reply:
Added a constant in case we want to use it in other places, but we have to pull it in like other compose libs.